demofile-parsing/parse_demo.js

```diff
@@ -1,24 +1,119 @@
 const fs = require("fs");
 const demofile = require("demofile");
 const path = require("path");
+const Round = require('./Round.js');
 
-fs.readFile(path.resolve("/Users/tylerlam/Downloads/test.dem"), (err, buffer) => {
+const DEBUG = false;
+const demPath = "/Users/tylerlam/Desktop/demfiles/ence-vs-astralis-m2-inferno.dem";
+const streamGameStart = 595; // specific to the stream chosen. Must set this before running
+var demoGameStart = null;
+
+// maps seconds time of demofile to timestamp of video
+function secondsToTimestamp(seconds, demoStart, streamStart) {
+  demoStart = demoStart || 0;
+  seconds = seconds - demoStart + streamStart;
+  var hours = Math.floor(seconds/3600);
+  seconds = seconds%3600;
+  var mins = Math.floor(seconds/60);
+  seconds = Math.floor(seconds%60);
+  hours = hours < 10 ? `0${hours}` : hours;
+  mins = mins < 10 ? `0${mins}` : mins;
+  seconds = seconds < 10 ? `0${seconds}` : seconds;
+  return `${hours}h${mins}m${seconds}s`;
+}
+
+
+fs.readFile(path.resolve(b), (err, buffer) => {
```

Comment:
Everything's good if we could just change this "b" variable to "demPath" for consistency.

Reply:
Yee thats my bad 😬